tests/conftest.py

```
@@ -682,6 +682,92 @@ def _diagnose_threads() -> None:
+
+        def __enter__(self):
+            return self
+
+        def __exit__(self, exc_type, exc, tb):
+            pass
+
+        def set_attribute(self, key: str, value) -> None:
+            pass
+
+    class IsolatedMockLogfire:
+        def span(self, name: str, *args, **kwargs):
```

Ruff (0.14.7)

Unused method argument: `args`

(ARG002)